data/mods/gen2stadium/conditions.ts

```diff
@@ -0,0 +1,131 @@
+//Since Stadium 2 ignores stat drops while recalculating stats due to boosting moves, haze, or stat-lowering moves.
+//The status conditions which affect a targets stats (brn and par) have a volatile along with them to keep track of if their respective stat changes
+//should be factored in during stat calculations or not.
```

Comment:
```suggestion
/**
 * Since Stadium 2 ignores stat drops while recalculating stats due to boosting moves, haze,
 * or stat-lowering moves. The status conditions which affect a targets stats (brn and par) have
 * a volatile along with them to keep track of if their respective stat changes should be factored
 * in during stat calculations or not.
 */
```